cEP-0009.md

```diff
@@ -0,0 +1,181 @@
+| Metadata     |                                                          |
+| ------------ |----------------------------------------------------------|
+| cEP          | 9                                                        |
+| Version      | 0.1                                                      |
+| Title        | Extract and utilize information from project files       |
```

Comment:
It seems to be aligned properly in my text editor. 
![image](https://cloud.githubusercontent.com/assets/10217535/26442151/d35dc6ca-4122-11e7-93d4-875d7d84232e.png)